### PR TITLE
[SUPPORT] Adding new Support tile

### PR DESCRIPTION
**Pull request by jedecf**
New Support Tile with owner

### CODE REVIEW

File: content/support/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+pcx_content_type: navigation
+title: Support
+weight: 1
+---
+ 
+# Welcome to the Support section
+ 
+**Coming Soon!** Future site for all Cloudflare Knowledge Base articles. We are in the process of migrating all of our helpful  Knowldge Base resources to live here permanently. Currently, you should still go to https://support.cloudflare.com/hc/en-us for all Knowledge Base articles.
+
+After migration, we will be moving product-specific Knowledge Base content yo live within the Troubleshooting section of its relevant product area. Any content that spans multiple products, is related to Account and Billing, or is related to working with Support will remain in this section.
```
Comment (kodster28) [inline content/support/_index.md]:
```suggestion
After migration, we will be moving product-specific Knowledge Base content to live within the Troubleshooting section of its relevant product area. Any content that spans multiple products, is related to Account and Billing, or is related to working with Support will remain in this section.
```

Comment (kodster28) [inline content/support/_index.md]: I also don't know how true that statement is... I'd imagine we'd move Account/Billing stuff into https://developers.cloudflare.com/fundamentals/account-and-billing/ and similar thoughts with the content spanning multiple products.

File: data/support.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  url: /support/
+
+meta:
+  title: Cloudflare Support docs
+  description: Cloudflare Support focused documentation.
+  author: '@cloudflare'
+  image: /core-services-preview.png
+
+resources:
+  community: https://community.cloudflare.com/tags/c/reliability/7/none
```
Comment (kodster28) [inline data/support.yml]:
```suggestion
```

File: data/support.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  dashboard_link: https://dash.cloudflare.com/?to=/:account/:zone/dns
+  learning_center: https://www.cloudflare.com/learning/dns/what-is-dns/
```
Comment (kodster28) [inline data/support.yml]:
```suggestion
```

File: data/support.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+meta:
+  title: Cloudflare Support docs
+  description: Cloudflare Support focused documentation.
+  author: '@cloudflare'
+  image: /core-services-preview.png
+
+resources:
+  community: https://community.cloudflare.com/tags/c/reliability/7/none
+  dashboard_link: https://dash.cloudflare.com/?to=/:account/:zone/dns
```
Comment (kodster28) [inline data/support.yml]:
```suggestion
  dashboard_link: https://dash.cloudflare.com/?to=/:account/support
```

**Review by kodster28 — CHANGES_REQUESTED** [review]

File: content/support/_index.md (hunk 1 above)
Comment (jedecf) [reply content/support/_index.md]: we could, for this tile placeholder deploy, not have that whole paragraph at all.

**Review by kodster28 — APPROVED** [review]

**Pull request merged**